wikimedia/wikipedia-ios
# Watchlist – [2] Add accessory rows to `WKForm`'s selectable section configuration

**Phabricator:** https://phabricator.wikimedia.org/T356794 (partially)

### Notes

This is part of a series of PRs that adds the functionality to add new languages from the Watchlist 
Filter view. This PR adds a new "accessory" row that allows the presentation of arbitrary row content in the same section as the multi-select toggle based "Wikipedias" section.

### Test Steps
1. Open Watchlist
2. Open the Filter View
3. Tap the Add Language accessory row
4. Confirm the "Wikipedia languages" view controller is presented (replaced with actual language selection view controller in [next PR](https://github.com/wikimedia/wikipedia-ios/pull/4755))
5. Confirm you are returned to the Filter view when closing the presented view controller
